### ISSUE TITLE
Warning: Use of undefined constant DD_TRACE_VERSION

### ISSUE THREAD

**Issue by gmansilla** [DataDog/dd-trace-php#494]
This warning is being shown on version 0.29

PHP version: PHP 7.2.17-1+ubuntu18.04.1+deb.sury.org+3 (cli) (built: Apr 10 2019 10:50:57) ( NTS )

Steps to reproduce:

1. Execute a php-cli script like the one bellow

    $DD_TRACE_CLI_ENABLED=true php artisan queue:listen --tries=1

> Warning: Use of undefined constant DD_TRACE_VERSION - assumed 'DD_TRACE_VERSION' (this will throw an Error in a future version of PHP) in /vendor/datadog/dd-trace/src/DDTrace/Transport/Http.php on line 58

**Comment by tiggreen** [comment]
I also experience this issue. It seems like `DD_TRACE_VERSION` is not defined in the class as a constant. We might want to define it the same way it's done for other variables.

**Comment by SammyK** [comment]
Hey @gmansilla and @tiggreen! This issue can arise when the tracer is updated with Composer, but not updated at the extension level. The recommended approach is to [install the PHP tracer from a package](https://docs.datadoghq.com/tracing/setup/php/#install-the-extension) which keeps the extension in sync with the PHP code. If you install the tracer from a package, you can remove the Composer dependency to ensure there are no conflicts going forward.

Let us know if you run into any issues after upgrading from a package. :)

**Comment by gmansilla** [comment]
@SammyK You're right. 

A quick note for anyone who comes across this issue and also needs to do manual instrumentation:

Take a look at [this comment](https://github.com/DataDog/dd-trace-php/issues/301#issuecomment-465215280).

TL;DR: use dd_trace('your/class', 'yourMethod', function(...$args) {  ....   $this->yourMethod(); });
